lib/demo_mode/config.rb

```diff
@@ -73,7 +73,7 @@ def around_persona_generation(&block)
       if block
         @around_persona_generation = block
       else
-        @around_persona_generation ||= :call.to_proc
+        @around_persona_generation ||= ->(generator, options) { generator.call(options: options) }
```

Comment:
@smudge  Is this what you are referencing to? I can remove the around block with this.